apps/backoffice-v2/src/pages/Entity/components/Case/hooks/usePendingRevisionEvents/interfaces.ts

```diff
@@ -0,0 +1,6 @@
+export interface IPendingEvent {
+  workflowId: string;
+  documentId: string;
+  pendingEvent: string;
```

Comment:
I think the pending event is the entire object and not the specific property. This naming will likely result in instances of `pendingEvent.pendingEvent`.
```suggestion
  event: string; // or name
```